### PR TITLE
1480: Update all openjdk.java.net->openjdk.org

### PR DESCRIPTION
**Pull request by erikj79**
The Skara source tree contains a lot of references to openjdk.java.net. All of them need to be updated to the new URL openjdk.org. Some of these are essential for core functionality such as jcheck so this is quite urgent.

This patch is a simple search-replace.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.java.net/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace

### Issue
 * [SKARA-1480](https://bugs.openjdk.org/browse/SKARA-1480): Update all openjdk.java.net->openjdk.org

### Reviewers
 * [Magnus Ihse Bursie](https://openjdk.org/census#ihse) (@magicus - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/skara pull/1338/head:pull/1338` \
`$ git checkout pull/1338`

Update a local copy of the PR: \
`$ git checkout pull/1338` \
`$ git pull https://git.openjdk.org/skara pull/1338/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 1338`

View PR using the GUI difftool: \
`$ git pr show -t 1338`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/skara/pull/1338.diff">https://git.openjdk.org/skara/pull/1338.diff</a>

</details>

### CODE REVIEW

**Comment by erikj79** [comment]
Test comment.

File: bots/pr/src/main/java/org/openjdk/skara/bots/pr/LabelCommand.java [hunk 1]
```diff
@@ -36,7 +36,7 @@ public class LabelCommand implements CommandHandler {
 
     private static final Pattern argumentPattern = Pattern.compile("(?:(add|remove)\\s+)((?:[A-Za-z0-9_@.-]+[\\s,]*)+)");
     private static final Pattern shortArgumentPattern = Pattern.compile("((?:[-+]?[A-Za-z0-9_@.-]+[\\s,]*)+)");
-    private static final Pattern ignoredSuffixes = Pattern.compile("^(.*)(?:-dev(?:@openjdk.java.net)?)$");
+    private static final Pattern ignoredSuffixes = Pattern.compile("^(.*)(?:-dev(?:@openjdk.org)?)$");
```
Comment (rgiulietti) [inline bots/pr/src/main/java/org/openjdk/skara/bots/pr/LabelCommand.java:39]: The literal dot in `openjdk.org` inside the `Pattern` should probably be escaped
`private static final Pattern ignoredSuffixes = Pattern.compile("^(.*)(?:-dev(?:@openjdk\\.org)?)$");`
I didn't check if there are other occurrences, though.

Comment (magicus) [reply bots/pr/src/main/java/org/openjdk/skara/bots/pr/LabelCommand.java:39]: While you are technically correct, this was not done before either. I don't think we should combine a massive, but trivial, search-and-replace PR with additional hands-on-fixes like that.

**Review by magicus — APPROVED** [review]
LGTM

**Comment by magicus** [comment]
It seems like a lot of redundancy in the code base. Not sure what to do about it, though. We're not likely to change URL once more, so ...

**Pull request closed without merging**